locations/spiders/pollotropical.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+class QuickleesSpider(scrapy.Spider):
+    name = "pollotropical"
+    allowed_domains = ["https://www.pollotropical.com/"]
+    item_attributes = {
+        "brand": "Pollo Tropical",
+        "brand_wikidata": "Q3395356",
+        "country": "USA",
```

Comment:
This adds country to yielded POI. Please use ISO-2 code: "US" not "USA"
Also you duplicate this by setting country further down. One or the other but not both!